torchbiggraph/config.py

```diff
@@ -455,22 +458,21 @@ def __del__(self) -> None:
         self.sys_path.remove(self.config_dir.name)
         self.config_dir.cleanup()
 
-    def load_raw_config(self, path: str) -> Any:
+    def load_raw_config(self, path: str, overrides: Optional[List[str]] = None) -> Any:
         module_name = f"torchbiggraph_config_{uuid.uuid4().hex}"
         shutil.copyfile(path, os.path.join(self.config_dir.name, f"{module_name}.py"))
         importlib.invalidate_caches()
         module = importlib.import_module(module_name)
         raw_config = module.get_torchbiggraph_config()
-        return raw_config
+        config_with_overrides = override_config_dict(raw_config, overrides)
+        return config_with_overrides
 
     def load_config(
         self,
         path: str,
         overrides: Optional[List[str]] = None,
     ) -> ConfigSchema:
         config_dict = self.load_raw_config(path)
```

Comment:
You forgot to forward the overrides.

Reply:
I moved the overrides to `load_raw_config`

Reply:
Yes, and you need to forward the `overrides` argument that was given to this function to that one.